test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Visitors/DictionaryObjectTypeVisitorTests.cs

```diff
@@ -56,6 +58,7 @@ public void Given_Type_When_IsVisitable_Invoked_Then_It_Should_Return_Result(Typ
         }
 
         [DataTestMethod]
+        [DataRow(typeof(Dictionary<object, string[]>), false)]
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.